giskard/scanner/robustness/text_transformations.py

```diff
@@ -150,10 +149,11 @@ def make_perturbation(self, text):
 class TextLanguageBasedTransformation(TextTransformation):
     needs_dataset = True
 
-    def __init__(self, column):
+    def __init__(self, column, rng_seed=None):
```

Comment:
Can you just set the `rng_seed` to a default value here? Because in the scan we then call `MyTextTransformation(column="xyz")`. Let’s make it deterministic by default.